spec/inject-challenge_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'inject-challenge'
+
+describe MyArray do
+  describe '#attempt_at_inject - without initial value argument' do
+    before do
+      subject.concat([1,2,3,4,5])
+    end
+
+    it 'behaves like inject { |r,e| r*e }' do
+      expect(subject.attempt_at_inject { |r, e| r*e }).to be( subject.inject { |r, e| r*e })
```

Comment:
Line is too long. [92/80]<br>Surrounding space missing for operator '*'.<br>Space inside parentheses detected.